### PR TITLE
Fix negative jitter

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.atlassian.performance.tools.jvmtasks.api
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.time.Duration.ofSeconds
+
+class JitterBackoffTest {
+
+    @Test
+    fun shouldYieldPositiveTime() {
```
Comment (wyrzyk) [inline src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt:12]: Isn't it flaky? Do you think it's worth to inject a Random to make the tests more predictable?

**Review by mzyromski-atlassian — DISMISSED** [review]

**Comment by wyrzyk** [comment]
I see you keep using separate commits for tests and the fix. With this approach, it's easier to show the change fixes a red build. OTOH we'll have build-breaking commits in the git history. It's also not a big deal as we can run a single test with git-bisect.
So this approach favours PRs over bisect investigation. TBH I don't recall running bisect on JPT project. I guess it's a good choice.

**Review by wyrzyk — DISMISSED** [review]

File: src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt (hunk 1 above)
Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt:12]: Good suggestion

**Comment by dagguh** [comment]
In JPT repos we used red commits often, we'll have to address that problem during bisecting anyway. So I want to at least preserve the value, since we already paid the cost.
I think red-green-refactor commits can still be bisectable. It's all up to the command we run. A `./gradlew build` will trip over the red ones. But `./gradlew test --tests TheTestToBisect` will work fine.

File: src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt (hunk 1 above)
Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/JitterBackoffTest.kt:12]: - [x] seed it

**Review by wyrzyk — APPROVED** [review]

**Review by mzyromski-atlassian — APPROVED** [review]

**Pull request merged**